calico_containers/adapter/netns.py

```diff
@@ -217,11 +217,12 @@ def reinstate_endpoint(cpid, old_endpoint, next_hop_ips,
     if_name = old_endpoint.if_name
     net = nets.pop()
     new_endpoint = set_up_endpoint(ip=net.ip,
+                                   hostname=HOSTNAME,
```

Comment:
should we just be passing in the hostname into this function, and using that.  It'll all amount to the same thing I believe, but we'd push the hostname determination out the powerstrip which invokes this function.